Guides/Building_AOSP.txt

```diff
@@ -126,6 +126,12 @@ $ prebuilts/misc/linux-x86/ccache/ccache -M 50G (or however much you want).
 Run this command if you used the automatic method of setup above
 $ ccache -M 50G
 
+Since Q, the way of using ccache has changed:
+$ export USE_CCACHE=1
+$ export CCACHE_EXEC=$(which ccache)
+$ ccache -M 50G
```

Comment:
Given that we set `CCACHE_EXEC` to the full path to `ccache`, it seems wise to use `${CCACHE_EXEC}` here or change `CCACHE_EXEC` to just `ccache`.